src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-10-8-0-320.mdx

```diff
@@ -0,0 +1,43 @@
+---
+subject: PHP agent
+releaseDate: '2023-03-22'
+version: 10.8.0.320
+downloadLink: 'https://download.newrelic.com/php_agent/archive/10.8.0.320'
+features: ['Healthcheck daemon log level']
```

Comment:
```suggestion
features: ['Add healthcheck daemon log level']
```
The Agent Metadata team suggests writing Release Notes metadata fields in the imperative.